src/Makefile

```diff
@@ -69,6 +72,9 @@ config-i386: config-gen
 	@cd kernel/include/ ; rm -f chal ; ln -s ../../platform/i386/chal/ chal
 	@echo "i386" > PLATFORM_ID
 
+run:
+	@cd ../transfer && echo "Running" $(RUNSCRIPT) && ./qemu.sh $(RUNSCRIPT)
```

Comment:
We likely want a default value for `$(RUNSCRIPT)` so that failure to provide it will be more obvious.

Reply:
this requires an update to the documentation in https://github.com/gwsystems/composite/blob/ppos/doc/README.md

Reply:
@gparmer to be honest, I tried to make it fail with an error when no runscript was provided, but it was beyond my makefoo. How would I do that?